staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -47,36 +50,65 @@ type openAPIAggregator struct {
 	// provided for dynamic OpenAPI spec
 	openAPIService *handler.OpenAPIService
 
-	// Aggregator's OpenAPI spec (holds apiregistration group).
-	aggregatorOpenAPISpec *spec.Swagger
+	contextMapper request.RequestContextMapper
 
-	// Local (in process) delegate's OpenAPI spec.
-	inProcessDelegatesOpenAPISpec *spec.Swagger
+	openAPIAggregationController *OpenAPIAggregationController
+}
 
-	contextMapper request.RequestContextMapper
+func (s *openAPIAggregator) addLocalSpec(spec *spec.Swagger, localHandler http.Handler, name, etag string) {
+	localApiService := apiregistration.APIService{}
```

Comment:
The same comment as before. It would look like this with a new import if I move Name: name inside:

```golang
import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
...
	localApiService := apiregistration.APIService{
		ObjectMeta: metav1.ObjectMeta {
			Name: name,
		},
	}
```
I prefer two existing to line of code with no extra import.